packages/firestore/test/unit/util/misc.test.ts

```diff
@@ -53,3 +53,216 @@ describe('FieldMask', () => {
     );
   });
 });
+
```

Comment:
Please move all the following classes into `describe('CompareUtf8Strings', () => {`